### PR TITLE
test: add test cases for `AvatarCard`

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[DET-7896]

## Test Plan
- Check test cases are covered enough

## Commentary (optional)
I added storybook as well.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

[DET-7896]: https://determinedai.atlassian.net/browse/DET-7896?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hamidzr — APPROVED** [review]
LGTM. thansk for testing these. this might need to be on upstream to pass the integration test.

File: webui/react/src/shared/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const Avatar: React.FC<Props> = (
   if (large) classes.push(css.large);
   if (className) classes.push(className);
   const avatar = (
-    <div className={classes.join(' ')} id="avatar" style={style}>
+    <div className={classes.join(' ')} data-testid="avatar-element" id="avatar" style={style}>
```
Comment (hamidzr) [inline webui/react/src/shared/components/Avatar/Avatar.tsx:53]: nit: is the change just to facilitate the tests? I'd guess we had added `id` for the same purpose. maybe we don't need both?

Comment (keita-determined) [reply webui/react/src/shared/components/Avatar/Avatar.tsx:53]: oh yeah. I couldn't see the `id` 😄

Comment (hamidzr) [reply webui/react/src/shared/components/Avatar/Avatar.tsx]: 👍🏼  I don't know which one is better to keep/use.

Comment (keita-determined) [reply webui/react/src/shared/components/Avatar/Avatar.tsx]: I think `id` is better in this case? https://stackoverflow.com/questions/55151142/react-testing-library-why-use-test-id

**Pull request merged**